2023/09.md

```diff
@@ -10,6 +10,20 @@
 - **Location**: TOKYO, JAPAN
 - **Attendee information**: https://github.com/tc39/Reflector/issues/499
 
+```mermaid
+---
+displayMode: compact
+---
+gantt
+    title 98th meeting
+    dateFormat  YYYY MM DD HH mm Z
```

Comment:
```suggestion
```

this is supposed to represent the date format you are inputting, and this doesn't match.  might as well just use the default iso-8601